tasks/scratch_release.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- name: "Tito release scratch"
+  command: "tito release {{ releasers | join(' ') }} --scratch"
```

Comment:
Similarly different -- could the --test or --scratch option be a variable? I realize this may not matter moving to obal

Reply:
Yeah, I started looking into it and had a half-written task for it, but with the decision to switch to obal I decided to just commit what I had been using currently as a proof of concept without refinement.